### DISCUSSION TITLE
How mdx v2 works ?

### DISCUSSION THREAD

**Discussion by johackim** [mdx-js/mdx#1861] [Q&A] [answered]
Hello, I try to use the v2 but I don't know how it works.

[Here an example on codesanbox](https://codesandbox.io/s/mdx-v2-67py0?file=/src/App.js)

Any help ?

Thanks !

**Comment by ChristianMurphy** [comment, marked as answer]
Here are several examples of using MDX 2
- [MDX Loader with Next.js](https://codesandbox.io/s/github/mdx-js/.github/tree/main/sandbox-templates/mdx-loader-next)
- [MDX Evaluate with Node.js](https://codesandbox.io/s/github/mdx-js/.github/tree/main/sandbox-templates/mdx-evaulate-node)
- [MDX Evaluate with Create React App](https://codesandbox.io/s/github/mdx-js/.github/tree/main/sandbox-templates/mdx-evaluate-create-react-app)

**Reply by johackim** [reply]
Awesome thanks @ChristianMurphy  !
